Codo-Baggins/covid_comparisons
# Feature/app setup

## What's this PR do?
- This PR builds out the main structure for all of the components that will be used in the application.  

## Where should the reviewer start?
- Start at App.tsx and then from there go to USAState.tsx, ComparisonContainer.tsx, and then Stat.tsx.

## How should this be manually tested?
- Clone down the repo
- `cd` into the cloned down directory
- Run `npm start` 
- Navigate to the localhost URL provided in the terminal
- Observe the content that is displayed on the URL

## Issues Resolved
- #1
